src/quote/swap/adapters/uniswap/index.ts

```diff
@@ -43,9 +46,9 @@ export class UniswapSwapQuoteProvider implements SwapQuoteProvider {
       throw new Error('Error - either input or output amount must be set')
     }
 
-    const weth = getTokenData('WETH', chainId)
-    const inputTokenData = getTokenDataByAddress(inputToken, chainId)
-    const outputTokenData = getTokenDataByAddress(outputToken, chainId)
+    const weth = getTokenByChainAndSymbol(chainId, 'WETH')
```

Comment:
same as above, with a proper chainId this returns WETH for sure, and the check in the if is unnecessary

Reply:
Since we can not control what chainId will be entered (by external clients/partners), I'd prefer this to fail further down below.